### DISCUSSION TITLE
Unhandled Promise Rejection: Cannot run updater on this Linux package. Currently only an AppImage can be updated.

### DISCUSSION THREAD

**Discussion by prajwalscodestack** [tauri-apps/tauri#6201] [Q&A]
I am getting this error in the console.
Also after running the app image of Tauri application, it is not installing the updates and not showing any notifications. 
But the server received the request.
please help

**Comment by FabianLars** [comment]
> I am getting this error in the console.

In what context? `tauri dev`? or when you run the binary or .deb file? in these cases this is intentional since the updater indeed only supports the appimage format.

> Also after running the app image of Tauri application, it is not installing the updates and not showing any notifications.

That's a bit tricky to triage. I guess for a start you could modify your main.rs file to log updater events. something like this
```rs
fn main() {

  let app = tauri::Builder::default()
    .build()
    .expcet("error building tauri app");
  app.run(|_app_handle, event| {
    if let tauri::RunEvent::Updater(event) = event {
      dbg!(event);
    }
  }

}
```
(you need to start the appimage from inside a terminal so that you can see the log output)
